pkg/trace/api/responses.go

```diff
@@ -46,6 +47,17 @@ func httpDecodingError(err error, tags []string, w http.ResponseWriter) {
 		errtag := "payload-too-large"
 		msg = errtag
 	}
+	// is it a timeout?
```

Comment:
These inline comments can be removed. I don't think the code is complex enough to warrant clarification.